cmd/common.go

```diff
@@ -41,3 +46,34 @@ func parseProfile(s string) (map[string]string, error) {
 	}
 	return params, nil
 }
+
+func getSpannerIds(ctx context.Context, targetProfile TargetProfile, now time.Time, driverName string, out *os.File) (string, string, string, error) {
```

Comment:
nit: This function refers to spannerIDs differently from this [spannerID](https://github.com/cloudspannerecosystem/harbourbridge/blob/master/internal/mapping.go#L162). Should be fine functionally but wondering if we can think of a better name? Nothing comes to my mind right now.

Reply:
That's a good find. I have renamed it to getResourceIds since each of project, instance and database are various resources.